src/components/ImageView/RegionView/RegionViewComponent.tsx

```diff
@@ -473,7 +473,7 @@ export class RegionViewComponent extends React.Component<RegionViewComponentProp
             stage.scale({x: refFrameZoom, y: refFrameZoom});
             const origin = scale2D({x: this.props.width / 2, y: this.props.height / 2}, 1 - refFrameZoom);
             const centerMovementCanvas = scale2D({x: refCenterMovement.x, y: -refCenterMovement.y}, refFrameZoom);
```

Comment:
We could combine the `scale2D` functions: `const centerMovementCanvas = scale2D({x: refCenterMovement.x, y: -refCenterMovement.y}, refFrameZoom / devicePixelRatio)`

Reply:
revised.